### PR TITLE
Sensors fix with sampling

### PR DESCRIPTION
**Pull request by g7**

### CODE REVIEW

File: sparse/usr/bin/droid/iio_increase_sampling_frequency.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+error() {
+	echo "E: ${@}"
+	exit 1
+}
+
+# systemd is dumb and doesn't support Restart=on-failure on oneshot scripts...
+for try in $(seq 5); do
+
+	sampling_frequency_file=$(find /sys/bus/iio/devices -type f -name in_accel_sampling_frequency -follow 2> /dev/null | head -n1)
```
Comment (piggz) [inline sparse/usr/bin/droid/iio_increase_sampling_frequency.sh:11]: What about the frequency of the other devices?

Comment (g7) [reply sparse/usr/bin/droid/iio_increase_sampling_frequency.sh:11]: Wondering what the impact would be - ALS is not super fast right now but still it does the job decently. Not sure about the other sensors as I haven't tested them extensively...

What do you think?

Comment (piggz) [reply sparse/usr/bin/droid/iio_increase_sampling_frequency.sh:11]: Ok, how about we commit this and monitor the others?

Comment (g7) [reply sparse/usr/bin/droid/iio_increase_sampling_frequency.sh:11]: That's fine for me :+1:

**Pull request merged**